src/kubectl-helm-minikube/install.sh

```diff
@@ -15,6 +15,7 @@ rm -rf /var/lib/apt/lists/*
 KUBECTL_VERSION="${VERSION:-"latest"}"
 HELM_VERSION="${HELM:-"latest"}"
 MINIKUBE_VERSION="${MINIKUBE:-"latest"}" # latest is also valid
+TOOLS_PATH="${TOOLS_PATH:-"/usr/local/bin"}" # latest is also valid
```

Comment:
```suggestion
INSTALL_TOOL_PATH="${INSTALLTOOLPATH:-"/usr/local/bin"}"
```

Would also need to rename all other instances of `TOOLS_PATH`